shell.nix

```diff
@@ -28,6 +28,7 @@ in
       electron
       pkg-config
       pixman cairo giflib libjpeg libpng librsvg pango            # build dependencies for node-canvas
+      shellcheck
```

Comment:
How does `shellcheck` work here? I do not see any reference to it.

Reply:
There is none, I should have mention it in the description, sorry. The CI failed on checking the shell script when I first created this PR. I wanted to reproduce locally but the `shellcheck` was missing. As the `shell.nix` should contain everything needed for the project, I added it.